src/App.js

```diff
@@ -1,26 +1,12 @@
-import logo from './logo.svg';
 import React from 'react';
-import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
     <div className="App">
```

Comment:
- [ ] Hello Also the **App.js** should be class-based components.

![Capture web_20-10-2021_125212_gitlab com](https://user-images.githubusercontent.com/61562626/138087644-eee9e69e-28fa-403c-834a-572267934e0e.jpeg)